guides/common/modules/proc_importing-a-subscription-manifest-into-satellite-server.adoc

```diff
@@ -19,11 +19,8 @@ endif::[]
 .Procedure
 
 . In the {Project} web UI, ensure the context is set to the organization you want to use.
-
 . Navigate to *Content* > *Subscriptions* and click *Manage Manifest*.
-
 . In the Manage Manifest window, click *Browse*.
-
```

Comment:
I do not see these newlines between steps as an issue. Especially in the procedures where some steps have code blocks in them. Removing them is not an issue either so I will merge as is.